oss_src/unity/python/sframe/test/test_gl_pickler.py

```diff
@@ -9,8 +9,8 @@
 from sframe.util import cloudpickle
 
 import sframe as gl
```

Comment:
This is no good too right?
it can't be import sframe.
See the other sframe/tests for how to do this.